Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -54,6 +54,9 @@
     <PackageReference Include="Xamarin.Build.TypeRedirector" Version="0.1.2-preview" PrivateAssets="all" />
     <Compile Include="**\*.shared.cs" />
     <Compile Include="**\*.shared.*.cs" />
+    <Compile Include="Contacts\Contacts.macos.cs">
+      <SubType></SubType>
+    </Compile>
```

Comment:
Here is bad boy part 2

Reply:
I removed this lines on VS Windows, compiled Android and transferred project to my MacBook and it started working. Magic

Thanks